packages/cli/src/commands/tile/deterministic.ts

```diff
@@ -19,16 +19,17 @@ export default class DeterministicTile extends Command<
     },
   ]
 
+  static flags = {
+    ...Command.flags,
+  }
   async run(): Promise<void> {
     this.spinner.start('Loading stream...')
 
     try {
       const tile = await TileDocument.deterministic(this.ceramic, this.args.metadata)
-      this.spinner.succeed(`Loaded stream ${tile.id.toString()}.`)
+      this.spinner.succeed(`Created tile ${tile.id.toString()}.`)
```

Comment:
I actually think it's better to tread the `deterministic` command more as a read than a write (though it can kind of be either)

Reply:
Fair, updated to `Loaded stream ${tile.id.toString()}`